src/Library/demos/Text Colors/main.py

```diff
@@ -0,0 +1,60 @@
+# Pango is a text layout library. It can e.g. be used for formatting text
+# https://gjs-docs.gnome.org/pango10~1.0/
+import gi
+
+gi.require_version("Gtk", "4.0")
+gi.require_version("Pango", "1.0")
```

Comment:
Specifying the version for Pango is not necessary, there is only one version.

Reply:
Oh sorry, I wasn't aware Pango 2 was a thing. In that case we should keep this :+1: 
See my comment here: https://github.com/sonnyp/Workbench/pull/732#discussion_r1382240886

Reply:
See also #738 - I'll think about / do some research if specifying the Pango version makes sense, but in doubt we keep it.